tasks/check-data-keys/check-data-keys.yaml

```diff
@@ -24,52 +29,46 @@ spec:
       description: The workspace where the data JSON file resides
   steps:
     - name: check-data-keys
-      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      image: quay.io/konflux-ci/release-service-utils@sha256:b8a4a4ff239d7789029a7bbd8c89d41453107d4391c1aaf7ea158f343d5624f6
```

Comment:
We actually should use sha tags everywhere, but we don't we use the commit tag so can you keep it consistent?

Reply:
I will create a ticket for it .... fixed it for time being